components/tyk-dashboard/README.md

```diff
@@ -0,0 +1,91 @@
+## Tyk Dashboard
+[Tyk](https://tyk.io) is an open source Enterprise API Dashboard, supporting REST, GraphQL, TCP and gRPC protocols.
+
+Tyk Dashboard is provided ‘Batteries-included’, with no feature lockout. Enabling your organization to control who accesses your APIs, when they access, and how they access it. Tyk Dashboard can also be deployed as part of a larger Full Lifecycle API Management platform Tyk Self-Managed which also includes Management Control Plane, Dashboard GUI and Developer Portal.
+
+[Overview of Tyk Dashboard](https://tyk.io/docs/tyk-dashboard/)
+
+## Introduction
+This chart defines a standalone Tyk Dashboard component on a [Kubernetes](https://kubernetes.io/) cluster using the [Helm](https://helm.sh/) package manager.
+
+For typical usage, we recommend using following umbrella charts:
+* For Tyk Open Source, please use [tyk-oss](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-oss)
+* For Tyk Pro, please use either [tyk-mdcb-data-plane](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-mdcb-data-plane) or [tyk-single-dc](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-single-dc)
+
+[Learn more about different deployment options](https://tyk.io/docs/apim/)
```

Comment:
[Learn more about Tyk Licensing and Deployment models](https://tyk.io/docs/tyk-on-premises/licensing/)

Reply:
added